student.rb

```diff
@@ -1,7 +1,12 @@
+require_relative 'person'
+
 class Student < Person
-  def initialize(age, classroom, parent_permission: true, name: 'Unknown')
-    super(age, parent_permission, name)
+  attr_accessor :name, :classroom
+
+  def initialize(age, classroom, name: 'Unknown')
+    super(age, name: name)
     @classroom = classroom
+    classroom.students.push(self) unless classroom.students.include?(self)
```

Comment:
- [ ] Nice adding student to the classroom automatically but it should be in the classroom method like the following 
```suggestion
    def classroom=(classroom)
      @classroom = classroom
      classroom.students.push(self) unless classroom.students.include?(self)
  end

```
that way your student can have many classrooms and the same for a classroom can have many students
and also will not return an error when there is no classroom

Reply:
Okay I have done it